docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1227,3 +1229,62 @@ FOR EACH ROW
 EXECUTE FUNCTION check_and_modify_val();
 
 ```
+
+---
+
+### Unsupported query construct: Advisory locks
```

Comment:
We don't need to add this prefix `Unsupported query construct:`, the feature heading can just be `Advisory locks`, `System Columns` and `XML Functions`